modyn/supervisor/internal/triggers/amounttrigger.py

```diff
@@ -16,8 +18,11 @@ def __init__(self, config: DataAmountTriggerConfig):
 
         assert self.data_points_for_trigger > 0, "data_points_for_trigger needs to be at least 1"
 
+    @override
```

Comment:
so far we don't use this decorator in the codebase. any reason to introduce it now here?

Reply:
For mypy to tell me when I forgot to update some signature of a function😅 The decorator is afaik only typing sugar, and doesn't affect runtime behaviour.